consensus/src/experimental/tests/execution_phase_tests.rs

```diff
@@ -34,7 +34,8 @@ fn add_execution_phase_test_cases(
 ) {
     let genesis_qc = certificate_for_genesis();
     let (signers, _validators) = random_validator_verifier(1, None, false);
-    let block = Block::new_proposal(Payload::empty(), 1, 1, genesis_qc, &signers[0], Vec::new());
+    let block =
+        Block::new_proposal(Payload::empty(), 1, 1, genesis_qc, &signers[0], Vec::new()).unwrap();
```

Comment:
Can you mark this `add_execution_phase_test_cases` function as test-only via `#[cfg(test)]` to ensure it's not called anywhere else and panic?

Reply:
This is already covered by the module